lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -342,12 +342,10 @@ def gem_version_compare(first, second)
                      curl --cacert $SSL_CA_CERT https://smart-proxy.example.com:8443"
           end
           def foreman_server_ca_cert
-            if File.exist?(Setting[:ssl_ca_file])
-              File.read(Setting[:ssl_ca_file])
-            else
-              msg = N_("SSL CA file not found, check the 'SSL CA file' in Settings > Authentication")
-              raise Foreman::Exception.new(msg)
-            end
+            File.read(Setting[:ssl_ca_file])
+          rescue => e
+            msg = N_("#{e.message}, check the 'SSL CA file' in Settings > Authentication")
```

Comment:
You can't use variables in translations. You have to use format strings.

Reply:
Just for reference, there is quite good tips we have in hammer: https://github.com/theforeman/hammer-cli/blob/master/doc/i18n.md#translation-tips

Reply:
Hmm, for some reason I thought that `N_` are not being translated so there is no need to handle variables. Anyway fixed now.